site/gatsby-site/src/utils/checklists.js

```diff
@@ -42,47 +42,41 @@ const statusColor = (status) => riskStatusFeatures[checkedRiskStatus(status)].co
 
 const abbreviatedTag = (tag) => tag.replace(/^.*:/g, '');
 
-const risksEqual = (risk1, risk2) => {
-  if (risk1.generated != risk2.generated) {
-    return false;
-  }
-  if (risk1.generated && risk2.generated) {
-    return risk1.title == risk2.title;
-  }
-  if (!risk1.generated && !risk1.generated) {
-    return risk1.id == risk2.id;
-  }
-};
-
 const Label = (props) => (
   <label {...{ ...props, className: `mb-1 block ${props.className || ''}` }}>
     {props.children}
   </label>
 );
 
-const emptyRisk = (properties) => ({
-  id: generateId(),
-  title: 'Untitled Risk',
-  tags: [],
-  risk_status: 'Not Mitigated',
-  risk_notes: '',
-  severity: '',
-  likelihood: '',
-  precedents: [],
-  touched: false,
-  generated: true,
-  startClosed: false,
-  ...(properties || {}),
-});
+const tagsIdentifier = (risk) => (
+  risk.tags.join("___")
+);
 
 const removeTypename = (obj) => {
   const replaced = JSON.stringify(obj).replace(/"__typename":"[A-Za-z]*",/g, '');
 
   return JSON.parse(replaced);
 };
 
-const exportJson = (checklist) => {
-  const json = JSON.stringify(checklist).replace(/"__typename":"[A-Za-z]*",/g, '');
+const joinManualAndGeneratedRisks = (manualRisks, generatedRisks) => {
+
+  const manualRiskTagIds = manualRisks.map(risk => tagsIdentifier(risk));
+
+  const unannotatedGeneratedRisks = generatedRisks.filter(
+    generatedRisk => !manualRiskTagIds.includes(tagsIdentifier(generatedRisk))
+  );
+
+  return manualRisks.concat(unannotatedGeneratedRisks);
+};
+
+const exportJson = (checklist, generatedRisks) => {
+  
+  generatedRisks ||= [];
+
+  const json = JSON.stringify({ 
+    ...checklist, 
+    risks: joinManualAndGeneratedRisks(checklist.risks, generatedRisks) 
+  }).replace(/"__typename":"[A-Za-z]*",/g, '');
```

Comment:
In this case, it's just because it's clutter and I don't think most users would want it. In other places we were removing `__typename` because it used to be the case that including `__typename`s in an updated value would cause a schema validation error. That's no longer true, so I took it out everywhere but here.